c2corg_ui/static/js/style/floatbuttons.js

```diff
@@ -0,0 +1,32 @@
+goog.provide('app.floatButtonResponsive');
+
+goog.require('app');
+
+/**
+ * 
+ * @export
+ */
+app.floatButtonResponsive = function() {
+  // This code will watch the Title height and adapt the buttons position accordingly.
+  var detailTitleHeight = $('.view-details-title').height();
+  adaptFloatingButtons(detailTitleHeight);
+
+  // FIX if i copy a adaptFloatingButtons to setTimeout, closure says that parameter is undefined o.O
+  $(window).resize(function() {
+    var detailTitleHeight = $('.view-details-title').height();
+    setTimeout(function() {
```

Comment:
I think my previous comment about this got lost. The way it is done right now does not reduce the number of calls to the resize event listener. Let's say without `setTimeout` you get 1000 calls to your function during a resize action. With `setTimeout` you still get 1000 calls, but the first call is delayed 200ms. That's all. I guess you actually wanted to use debouncing (call the function only once during a timespan of 200ms)?

Maybe we can avoid `setTimeout` by using `matchMedia`?